### ISSUE TITLE
SnagIt PKG recipe failing

### ISSUE THREAD

**Issue by LukeyGee** [autopkg/bkerns-recipes#18]
I have run into the following error running your pkg recipe for SnagIt. Previous releases up to 2019.1.4 have been fine.

```
TechSmithURLProvider
TechSmithURLProvider: Found URL http://download.techsmith.com/snagitmac/releases/2000/snagit.dmg
TechSmithURLProvider: Found Version 20.0.0
URLDownloader
URLDownloader: Item at URL is unchanged.
URLDownloader: Using existing /Volumes/Data/AutoPkg/Cache/com.github.autopkg.kernsb.pkg.TechSmithSnagit/downloads/TechSmithSnagit.dmg
EndOfCheckPhase
CodeSignatureVerifier
CodeSignatureVerifier: Mounted disk image /Volumes/Data/AutoPkg/Cache/com.github.autopkg.kernsb.pkg.TechSmithSnagit/downloads/TechSmithSnagit.dmg
Error processing path '/private/tmp/dmg.kr8CMZ/Snagit 2019.app' with glob. 
Failed.
```

**Comment by WaveBirdDash** [comment]
#17 will address this issue.

**Comment by kernsb** [comment]
Addressed via #17 merge.
